api/src/main/resources/workbench.yaml

```diff
@@ -525,9 +525,10 @@ paths:
           required: true
       responses:
         "200":
-          description: A collection of subjects
+          description: A count of subjects
```

Comment:
So just to confirm: we are now returning and displaying simple counts from the cohort builder, rather than counts broken down by race / gender?

(Sounds fine to me -- just checking.)

Reply:
Correct. We need to do something for the charts to render(gender and age), but we will have to create another story for that.

Reply:
OK. For that, I suspect you could just change COUNT_SQL_TEMPLATE to be:

select gender_concept_id, race_concept_id, count(distinct person_id) as count
...
group by gender_concept_id, race_concept_id

and either return the results as-is or use them to calculate totals for each race or gender.

Reply:
yep.. that sounds about right